hack/go-install.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -uo pipefail
+
+DEST_DIR=${1:-}
+CMD=${2:-}
+GO_INSTALL_LOC=${3:-}
+
+if [[ ("$DEST_DIR" == "" || "$CMD" == "" || "$GO_INSTALL_LOC" == "") ]]; then
+    echo "Usage: $0 DEST_DIR CMD GO_INSTALL_LOC"
+    exit 1
+fi
+
+set +e
+CMD_PATH=$(command -v "$CMD")
+set -e
```

Comment:
What if we make it more explicit, for example:

```suggestion
CMD_PATH=$(command -v "$CMD" || true)
```

Reply:
Yep adjusted as suggested.